src/evm-canister-client/src/client.rs

```diff
@@ -826,4 +826,28 @@ impl<C: CanisterClient> EvmCanisterClient<C> {
     pub async fn is_inspect_message_disabled(&self) -> CanisterClientResult<bool> {
         self.client.query("is_inspect_message_disabled", ()).await
     }
+
+    /// Returns the current transaction processing interval in seconds
+    pub async fn get_transaction_processing_interval(&self) -> CanisterClientResult<u64> {
```

Comment:
please name it exactly as the corresponding evmc endpoint. i.e.: `get_transaction_processing_interval_secs`